merlin/main.py

```diff
@@ -502,29 +505,32 @@ def print_info(args: Namespace):
 
 def config_merlin(args: Namespace):
     """
-    CLI command to set up the default Merlin configuration.
+    CLI command to manage Merlin configuration files.
 
-    This function initializes the configuration app.yaml file that's
-    necessary to connect Merlin to a central server. If the output
-    directory is not specified via the command-line arguments, it
-    defaults to the user's home directory under `.merlin`.
+    This function handles various configuration-related operations based on
+    the provided subcommand. It ensures that the specified configuration
+    file has a valid YAML extension (i.e., `.yaml` or `.yml`).
+
+    If no output file is explicitly provided, a default path is used.
 
     Args:
-        args: Parsed command-line arguments, which may include:\n
-            - `output_dir`: Path to the output directory for
-              configuration files. If not provided, defaults to
-              `~/.merlin`.
-            - `task_server`: Address of the task server for the
-              configuration.
-            - `broker`: Address of the broker service to use.
-            - `test`: Flag indicating whether to run in test mode.
+        args: Parsed command-line arguments.
     """
-    output_dir: Optional[str] = args.output_dir
-    if output_dir is None:
-        user_home: str = os.path.expanduser("~")
-        output_dir: str = os.path.join(user_home, ".merlin")
+    if not args.config_file.endswith((".yaml", ".yml")):
```

Comment:
while it's reasonable, `.YML` would be valid and fail. Shouldn't we raise the error when loading the yaml?

Reply:
@doutriaux1 does using the `.yml` extension not work when loading in yaml files? I would've thought it's fine

Reply:
oh my point was if the user pass a config name `CONFIG.YML` it would fail
 
and also that if they pass a valid yaml file named `config.txt` it would fail w/o trying to open it
 
so I was thinking than rather than using `endswith()` you might want to simply try to load the file via `yaml.safe_load()` these files are tiny so it shouldn't take long anyway.